test/ssl-tests/18-dtls-renegotiate.cnf.dat

```diff
@@ -123,7 +118,6 @@ foreach my $sctp ("No", "Yes")
             },
             test => {
                 "Method" => "DTLS",
-                "UseSCTP" => $sctp,
                 "HandshakeMode" => "RenegotiateServer",
                 "ResumptionExpected" => "No",
                 "ExpectedResult" => "Success"
```

Comment:
Then end the scope above the `next if` a few lines down from here, and change the next to an if clause to start a new scope with

```perl
if (!disabled("dtls1_2")) {